src/pages/roadmap/Section.tsx

```diff
@@ -29,32 +28,29 @@ export const Section: React.FC<SectionProps> = ({
   projects,
   className,
 }) => {
-  const isMobile = useIsMobile()
   return (
     <div className={clsx(styles.container, iconPosition === 'right' && styles.reverse, className)}>
       <div className={styles.iconWrapper}>
         <div className={styles.icon}>
-          <img
-            src={icon}
-            alt=""
-            style={isMobile ? { width: 184, height: 184 } : { width: iconSize.with, height: iconSize.height }}
-          />
+          <img src={icon} alt="" style={{ width: iconSize.with, height: iconSize.height }} />
         </div>
       </div>
       <div className={styles.detailWrapper}>
         <div>{title}</div>
         <div className={styles.divider} />
         <div className={styles.subtitle}>{subtitle}</div>
-        <div className={styles.descriptionText}>
-          {descriptionText} {/* TODO: add real href here */}
-          <StyledLink className={styles.link} href={''} colored>
-            {descriptionLink}
-          </StyledLink>
-        </div>
-        {projects.map(project => (
-          <div className={styles.project} key={project}>
-            {project}
+        {descriptionText.startsWith('Lorem') ? null : (
```

Comment:
This unique business logic implementation is a bit hidden, making it difficult for people to quickly discover its relationship with locales. Consider using more obvious markers:

1. "//description": "..."
1. "description": "\_\_hidden_this_description\_\_ ..."

Reply:
Prefix `__hidden__description__` was used as an annotation in commit https://github.com/Magickbase/nervos-official-website/pull/360/commits/c673c32c11668cc5d46c6eda893ba0feaf78c6f4